conf/www.rabbitmq.com-common

```diff
@@ -77,11 +77,23 @@
 
     Alias /blog /usr/share/wordpress
 
+    # Redirect manpages from old location to new one that does not have '.man'
+    # in the URL and may change section as well
+
+    # NB: special-case section 5 man pages (rabbitmq-env.conf) as they did not
+    # change section
+    RedirectMatch permanent ^/man/(.+)\.5\.man\.html /$1.5.html
+    RedirectMatch permanent ^/(.+)\.5\.man\.html /$1.5.html
+
+    # NB: every other man page changed from the old section to section 8
```

Comment:
They were changed from section 1 (general purpose commands) to 8 (service management commands). You don't  need to (and shouldn't) cover other sections: a system call manpage won't become a service management command.